package/MDAnalysis/analysis/pca.py

```diff
@@ -177,26 +234,6 @@ class PCA(AnalysisBase):
 
```

Comment:
we'll probably want a `versionchanged` entry here just to say that we are now storing results using `MDAnalysis.analysis.base.Results`.